docsbox/docs/utils.py

```diff
@@ -110,12 +112,40 @@ def get_pdfa_version(nodes):
             conformance = x.firstChild.nodeValue
     return part + conformance
 
+def store_file(data, filename, stream=False):
+    suffix = os.path.splitext(filename)[1] if filename else ""
+    with NamedTemporaryFile(delete=False, dir=app.config["MEDIA_PATH"], suffix=suffix) as tmp_file:
+        if stream:
+            for chunk in data.iter_content(chunk_size=128):
```

Comment:
A chunk size of 128 bytes is super small. These days it can safely be 8kb. Such small chunk size will slow the process down quite much.

Check all places where this is defined in the project and fix accordingly.